src/CLIParsing.h

```diff
@@ -14,7 +14,10 @@ static void CLIArgumentInit(QApplication& translateLocallyApp, QCommandLineParse
     parser.setApplicationDescription("A secure translation service that performs translations for you locally, on your own machine.");
     parser.addHelpOption();
     parser.addVersionOption();
-    parser.addOption({{"l", "list-models"}, QObject::tr("List available models.")});
+    parser.addOption({{"l", "list-models"}, QObject::tr("List locally installed models.")});
+    parser.addOption({{"a", "available-models"}, QObject::tr("Connect to the Internet and list available models. Only shows models that are NOT installed locally or have a new version available online.")});
+    parser.addOption({{"d", "download-model"}, QObject::tr("Connect to the Internet and download a model."), "output", ""});
+    parser.addOption({{"r", "remove-model"}, QObject::tr("Remove a model from the local machine. Only works for models managed with translateLocally."), "output", ""});
     parser.addOption({{"m", "model"}, QObject::tr("Select model for translation."), "model", ""});
     parser.addOption({{"i", "input"}, QObject::tr("Source translation text (or just used stdin)."), "input", ""});
     parser.addOption({{"o", "output"}, QObject::tr("Target translation output (or just used stdout)."), "output", ""});
```

Comment:
I'm a bit confused about these descriptions? Is the argument you pass it used as text, or is it supposed to be a file? From the code I gather it's the latter, but description is a bit unclear.